internal/backends/qt/qt_widgets/spinbox.rs

```diff
@@ -192,13 +198,13 @@ impl Item for NativeSpinBox {
                 }
                 MouseEvent::Moved { .. } => false,
                 MouseEvent::Wheel { delta_y, .. } => {
-                    if delta_y < 0. {
+                    if delta_y > 0. {
```

Comment:
Actually all the other style have the same behavior (scroll up decrease the value)

Reply:
Fixing that in https://github.com/slint-ui/slint/pull/5012